spec/domain/operations/individual/calculate_monthly_aggregate_spec.rb

```diff
@@ -281,6 +281,18 @@
         end
       end
 
+      context 'Gap in eligible months of 4months will not be considered and effective date is middle of month.' do
+        before do
+          hbx_enrollment.update_attributes(terminated_on: TimeKeeper.date_of_record - 1.day)
+          input_params = {family: family, effective_on: Date.new(current_year, 11, 10), shopping_fm_ids: hbx_enrollment.hbx_enrollment_members.pluck(:applicant_id), subscriber_applicant_id: hbx_enrollment.subscriber.applicant_id}
+          @result = subject.call(input_params)
+        end
+
+        it 'should return aptc amount based on eligible months' do
```

Comment:
Test coverage to check the returned aptc amount for the case where effective_on date is not the first of the month.